### ISSUE TITLE
Testing Accuracy Issue

### ISSUE THREAD

**Issue by leo19910821** [swathikirans/violence-recognition-pytorch#2]
i have run the code with your default parameters but i am getting only 50% accuracy on testing.... Can you please explain what could be the problem

![result1](https://user-images.githubusercontent.com/32150927/48130536-fadda880-e2c7-11e8-985a-b714ce9f46e2.png)

**Comment by swathikirans** [comment]
Which dataset are you using? The testing loss is too high. Seems like there's something wrong, perhaps the way in which you loaded the data

**Comment by leo19910821** [comment]
I am Using the hockeyFight Dataset and didn't change any code.

**Comment by swathikirans** [comment]
Are you sure that the frames are placed in the directory in the way described in the readme?

**Comment by stevve** [comment]
my result of accuacy is always be 0
![image](https://user-images.githubusercontent.com/12237266/53334469-e505f900-3933-11e9-9271-76df6fb6f535.png)
1.I place frame in this way (directory fights and nofights are in same level)

![image](https://user-images.githubusercontent.com/12237266/53333765-c0a91d00-3931-11e9-8f5c-3baf11bb0b70.png)

 is this correct way place dataset?

2. I did't split the trainset and testset ,all video are in one directory,
fights(contains 500 fights directory)
nofight(contains 500 nofights directory)
3 how you change video to frame ,just change all video into frame or 5 frame/s?

**Comment by swathikirans** [comment]
1. The structure of the directory shown seems to be correct
2. As a proof of concept, you can use the entire dataset as both train and test or create a function to perform the train/validation sets. I will try to update the repo with this function in the coming days, I am not sure when since I am quite busy with other works
3. I extracted all the frames from the video and the dataset loader will select the frames uniformly across time, depending on the number of frames set in the "seqLen" argument. The following code snippet shows how the frames are extracted:

```
import os
import glob
root_dir = './datasets/hockey/' # video folder
des_dir = root_dir + 'frames/' # frame folder
for i in range(500):
    fl = root_dir + 'noFights/no' + str(i+1) + '_xvid.avi'
    des_dir1 = des_dir + 'noFights/' + str(i+1)
    os.system('mkdir ' + des_dir1)
    os.system('ffmpeg -i ' + fl + ' ' + des_dir1 + '/frame%03d.jpg')
```
There was an error in the multi-crop evaluation code. I have corrected it and updated the repo. However, it should not affect the training results.

**Comment by swathikirans** [comment]
> I am Using the hockeyFight Dataset and didn't change any code.



> i have run the code with your default parameters but i am getting only 50% accuracy on testing.... Can you please explain what could be the problem
> 
> ![result1](https://user-images.githubusercontent.com/32150927/48130536-fadda880-e2c7-11e8-985a-b714ce9f46e2.png)

There was an error in the multi-crop evaluation. I have corrected it and updated the repo.

**Comment by jaiminkansara** [comment]
@swathikirans what is the output from this.? does this generate a file showing which test videos are fight (or no fight). or doest is generates labels over frames where fight is detected?
